### ISSUE TITLE
Can the hex file in nRF52840_dongle also be used in nRF52840 DK？

### ISSUE THREAD

**Issue by chenyixi1997** [kardia-as/nrf-zboss-ncp#7]
I am a newbie, can the hex file in nRF52840_dongle also be used in nRF52840 DK？

**Comment by chenyixi1997** [comment]
After trial, it doesn't seem to work

**Comment by DamKast** [comment]
No, because the dongle is using CDC ACM through the USB interface.
And I think the overlay for the dongle is different from the nRF52840 DK
